docs/product/sentry-basics/concepts/tracing/trace-view.mdx

```diff
@@ -4,47 +4,117 @@ sidebar_order: 30
 description: "Learn more about trace view, where you can drill down into the details of a single trace, allowing you to debug slow services, identify related errors, and find other bottlenecks."
 ---
 
-With the [trace](/product/sentry-basics/tracing/distributed-tracing) view, you can drill down into the details of a single trace and traverse every transaction in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks. There are multiple entry points to get to the trace view.
+The trace view is designed to be your one-stop-shop when debugging performance or errors. It gives you full context on what was happening when an error or performance issue occurred, all in one place. The waterfall trace view allows you to see everything that may have occurred during a trace, including errors, performance issues, and any profiles that may have been collected.
 
-## Entry Points
-
-- [Issue Details](/product/issues/issue-details/): In the issue detail, click "View Full Trace" below the event ID (paired with Trace Navigator).
-- [Event Details](/product/performance/event-detail/): In an event detail, click the trace ID on the top right (paired with Trace Navigator).
-- [Transaction Summary](/product/performance/transaction-summary/): In the transaction summary, click the trace ID in the event table.
-- [Query Builder](/product/discover-queries/query-builder/): In Discover, build a query with the trace column, then click the trace ID in the resulting event table.
+In addition, the trace view allows you to drill down into the details of a single trace so you can visualize the high level transactions and spans that took place within that trace. This makes debugging slow services, identifying related errors, and rooting out bottlenecks easer and faster without having to navigate around multiple pages of the Sentry product.
 
 ![Example of Trace View](./img/trace-view.png)
```

Comment:
Should we update or remove this screenshot? I see later we have a walkthrough that's based on the new view

Reply:
Yup, definitely!